components/home/Chat.tsx

```diff
@@ -116,6 +120,17 @@ export default function Chat() {
       };
       handleAsync();
     }
+    async function getPerms() {
+      const { data, error } = await getCurrentUserChannelPermissions(
+        supabase,
+        channelId
+      );
+
+      if (error) console.log(error);
+
+      setUserPerms(data);
+    }
+    getPerms();
```

Comment:
This should leverage realtime to reflect permission changes on the fly, otherwise the user has to leave/re-enter the channel to see the reflected changes

Reply:
realtime has been added